### ISSUE TITLE
Required field can't be empty

### ISSUE THREAD

**Issue by svanschu** [JoomTools/plg_fields_jtfileupload#18]
The upload field is empty if you're editing an existing article (file is already uploaded).
If the field is required the JS forbids to save the form if the field is empty, but we're allowing an empty field if there is a file already uploaded.

**Comment by svanschu** [comment]
already done because off set field to disabled if a file already is uploaded.

See #9

**Comment by svanschu** [comment]
It's here again

**Comment by webcomil** [comment]
same here, even if not set as reuireed, getting the  erroe meesge.
furthermore, cannot replace the current file

**Comment by svanschu** [comment]
Yes i need to look into it. currently setup my new development computer because the old broke down. Then I can fix it
